swig/cpp/examples/cpp_turing_rpc_example.cpp

```diff
@@ -56,9 +56,9 @@ class My_Callback:public sysrepo::Callback {
             auto output = holder->allocate(2);
 
             /* set 'output/step-count' leaf */
-            output->val(0)->set("/turing-machine:run-until/step-count", (uint64_t) 256, SR_UINT64_T);
+            output->val(0)->set("/turing-machine:run-until/step-count", (uint64_t) 256);
```

Comment:
Given that you're modifying these anyway, my recommendation is to use C++11-style construction of the numbers as well, i.e.:
```c++
foo->set("/some/xpath", uint64_t{123456});
```
Unlike the C-style cast which might be ambiguous in a C++ context (that's the reason why we have various `xxx_cast<>` statements), this one prevents errors when the value would overflow. These are examples, they should follow best practices.